data/migrations/20191907012229_action-table.js

```diff
@@ -0,0 +1,31 @@
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable('actions', function(tbl) {
+        
+      tbl.increments()
+      .unique()
```

Comment:
Using `.unique()` is redundant here, there will be no repeated ids with `.increments()`